### PR TITLE
Add first api call

### PR DESCRIPTION
**Pull request by jzhu-eth**

### CODE REVIEW

File: v2/portfolio/account_service.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package portfolio
```
Comment (jzhu-eth) [inline v2/portfolio/account_service.go:1]: cp file from delivery

File: v2/portfolio/request.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
+package portfolio
```
Comment (jzhu-eth) [inline v2/portfolio/request.go:1]: cp file from delivery

File: v2/portfolio/client.go [hunk 3]
```diff
@@ -0,0 +1,172 @@
+package portfolio
```
Comment (jzhu-eth) [inline v2/portfolio/client.go:1]: cp code from delivery

File: v2/portfolio/account_service.go (hunk 1 above)
Comment (jzhu-eth) [inline v2/portfolio/account_service.go:1]: I’m following the Go Binance code style, even though I don’t like it.

**Review by maxkaneco — APPROVED** [review]

**Review by maxkaneco — APPROVED** [review]

**Pull request merged**